### PR TITLE
initial site header version

### PR DESCRIPTION
**Pull request by pavel-kirpikau**

### CODE REVIEW

File: src/components/header/Header.css [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    align-content: stretch;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px white solid;
+    }
+
+.flex-item:nth-child(1) {
```
Comment (GitStearis) [inline src/components/header/Header.css:38]: I can go wrong, but `flex-item` classes could be removed without any harm. Please check the behaivor of the header without all `flex-item` styles.

File: public/assets/icons/email.svg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0"?>
```
[the remark this answers is not in the document]
Comment (pavel-kirpikau) [reply public/assets/icons/email.svg]: svg icons removed

File: src/components/header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import facebookIcon from '@iconify/icons-cib/facebook';
+import vkIcon from '@iconify/icons-cib/vk';
+import mailRu from '@iconify/icons-cib/mail-ru';
+import instagramIcon from '@iconify/icons-cib/instagram';
+import telegramIcon from '@iconify/icons-cib/telegram';
+import googlePlay from '@iconify/icons-cib/google-play';
+import bxSearch from '@iconify/icons-bx/bx-search';
+
+import './Header.css'
+
```
[the remark this answers is not in the document]
Comment (pavel-kirpikau) [reply src/components/header/Header.tsx:12]: updated according to TS styleguide

File: src/components/header/Header.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+
+function Header() {
+    return (
+      <div className='.header flex-container'>
+        <p className="logo-text flex-item:nth-child(1)">RADIO PLATO</p>
```
[the remark this answers is not in the document]
Comment (pavel-kirpikau) [reply src/components/header/Header.tsx]: redundant `nth-child` classes removed

File: src/components/header/Header.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+import googlePlay from '@iconify/icons-cib/google-play';
+import bxSearch from '@iconify/icons-bx/bx-search';
+
+import './Header.css'
+
+
+
+function Header() {
+    return (
+      <div className='.header flex-container'>
```
[the remark this answers is not in the document]
Comment (pavel-kirpikau) [reply src/components/header/Header.tsx]: class names updated to unique

File: src/components/header/Header.css (hunk 1 above)
Comment (pavel-kirpikau) [reply src/components/header/Header.css]: You are right. I removed all `flex-item` classes until needed for styling

**Pull request merged**